src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
```

Comment:
PullProofAI

The `ApiProperty` decorator is used from the `@nestjs/swagger` package to provide metadata for Swagger documentation. It seems like the `example` property is incorrectly defined as an array instead of a string. Let's correct this by providing a single string value for the `example`.

```diff
-     example: ['practice1', 'practice2'],
+     example: 'practice1, practice2',
```



<!-- This is an auto-generated comment by PullProofAI -->